src/components/atoms/ProgressBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import cx from 'classnames';
+import React from 'react';
+
+export type ProgressBarProps = {
+  totalsteps?: string;
+  stepsready?: string;
+  label?: string;
+};
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  totalsteps,
+  stepsready,
+  label,
+}) => {
+  const StepsReadyPercente = (Number(stepsready) / Number(totalsteps)) * 100;
+
+  return (
+    <div>
+      <span className="p-2.5 text-sm font-semibold text-slate-50">
+        {label} {stepsready}/{totalsteps}
+      </span>
+      <div className=" h-[8px] w-[150px] rounded-md bg-slate-100">
```

Comment:
Let's try to use the `h-1, h-2, h-3` options from tailwind when possible (https://tailwindcss.com/docs/height).
Same for width. 

But in this case, **width should be** `100%`. The parent (the one who call <ProgressBar/>) will be responsible to manage it's size and positioning in the page.